packages/contracts/docs/src/src/dollar/Diamond.sol/contract.Diamond.md

```diff
@@ -1,5 +1,5 @@
 # Diamond
-[Git Source](https://github.com/ubiquity/ubiquity-dollar/blob/565aaa6bed7cb481fd57c9fc6a7b1052ff2aa816/src/dollar/Diamond.sol)
+[Git Source](https://github.com/ubiquity/ubiquity-dollar/blob/0cae71618450aff584ed3369a18e2ba12900dc6b/src/dollar/Diamond.sol)
```

Comment:
Why are there so many `md` files changes? There's a separate docs generation [workflow](https://github.com/ubiquity/ubiquity-dollar/blob/development/.github/workflows/generate-doc.yml) that updates docs after each push to the `development` branch so there's no need update docs manually. Besides https://github.com/ubiquity/ubiquity-dollar/blob/0cae71618450aff584ed3369a18e2ba12900dc6b/src/dollar/Diamond.sol leads to unknown file so it seems that commit `0cae71618450aff584ed3369a18e2ba12900dc6b` does not exist.

Reply:
im not sure maybe because i had to force push some changes after i created the PR? i didnt do this intentionally